game-cards/SGgame.js

```diff
@@ -1201,9 +1201,91 @@ exports.commands = {
 		if (!p.bag.pokeballs[target[1]]) p.bag.pokeballs[target[1]] = 0;
 		p.bag.pokeballs[target[1]] += target[2];
 		Db.players.set(u.userid, p);
-		return this.sendReply(`${u.userid} has been given ${target[2]} ${target[1]}'s. They now have ${p.bag.pokeballs[target[1]]} ${target[1]}'s.`);
+		if (Rooms.get('staff')) {
+			Rooms.get('staff').add('|raw| <div class="broadcast-green">' + WL.name(user.name, true) + ' has given ' + target[2] + ' ' + target[1] + 's to ' + (u.userid) + '</div>').update();
+		}
+		this.sendReply(`${u.userid} has been given ${target[2]} ${target[1]}'s. They now have ${p.bag.pokeballs[target[1]]} ${target[1]}'s.`);
 	},
 	givepokeballshelp: ['/givepokeballs [user], [type], [amount] - Give a user pokeballs. Requires global @ & ~'],
+	tp: 'takepokeballs',
+	takepokeballs: function (target, room, user) {
+		// Allows mods+ to give more pokeballs during the alpha
+		if (!this.can('ban')) return;
+		target = target.split(',').map(part => {
+			return toId(part);
+		});
+		if (target.length < 3) return this.parse(`/help takepokeballs`);
+		let u = target[0] = Users(target[0]);
+		if (!u) return this.errorReply(`User "${target[0]}" not found.`);
+		 if (!['pokeball', 'greatball', 'ultraball', 'masterball'].includes(target[1])) return this.parse(`/help takepokeballs`);
+		if (target[1] === 'masterball' && !user.can('lockdown')) return this.errorReply(`Only Administrators may take masterballs.`);
+		target[2] = parseInt(target[2]);
+		if (isNaN(target[2]) || target[2] < 1 || target[2] > 100) return this.errorReply(`Pokeball amount must be a number between 1 and 100.`);
+		let p = Db.players.get(u.userid);
+		if (!p) return this.errorReply(`${u.userid} has not started SGgame and cannot be taken pokeballs at this time.`);
+		if (!p.bag.pokeballs[target[1]]) p.bag.pokeballs[target[1]] = 0;
+		p.bag.pokeballs[target[1]] -= target[2];
+		Db.players.set(u.userid, p);
+		if (Rooms.get('staff')) {
+		    Rooms.get('staff').add('|raw|<div class="broadcast-green"> ' + WL.nameColor(user.name, true) + ' has taken ' + target[2] + ' ' + target[1] + ' from ' + (u.userid) + '.').update();
+		}
+		this.sendReply(`${u.userid} has been taken ${target[2]} ${target[1]}'s. They now have ${p.bag.pokeballs[target[1]]} ${target[1]}'s.`);
+	},
+	takepokeballshelp: ['/takepokeballs [user], [type], [amount] - Take a users pokeballs. Require global @ & ~.'],
+
+	gs: 'givestone',
+	givestone: function (target, room, user) {
```

Comment:
Ill implement mega stones in an obtainable way somewhere on the map in V2, no need to have a command for giving/taking them.

Reply:
It was meant for v1 and before doing that, I didn't know v2 was already in development.